socorro/lib/prioritize.py

```diff
@@ -1,158 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Looks like this was missed when we refactored out the other griswolf code some years back. Good catch.